.gitignore

```diff
@@ -0,0 +1,111 @@
+# Compiled source #
+###################
+*.com
+*.class
+*.dll
+*.exe
+*.o
+*.so
+*.less
+*.handlebars
+
+# Packages #
+############
+# it's better to unpack these files and commit the raw source
+# git has its own built in compression methods
+*.7z
+*.dmg
+*.gz
+*.iso
+*.jar
+*.rar
+*.tar
+*.zip
+
+# Logs and databases #
+######################
+*.log
+*.sql
+*.sqlite
+
+# OS generated files #
+######################
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+
+# Windows image file caches #
+######################
+Thumbs.db
+ehthumbs.db
+
+# Folder config file #
+######################
+Desktop.ini
+
+# Recycle Bin used on file shares #
+######################
+$RECYCLE.BIN/
+
+# Windows Installer files #
+######################
+*.cab
+*.msi
+*.msm
+*.msp
+
+# Windows shortcuts #
+######################
+*.lnk
+
+# Images #
+######################
+#*.jpg
+#*.gif
+#*.png
+#*.svg
+*.ico
+
+# Video #
+######################
+*.wmv
+*.mpg
+*.mpeg
+*.mp4
+*.mov
+*.flv
+*.avi
+*.ogv
+*.ogg
+*.webm
+
+# Audio #
+######################
+*.wav
+*.mp3
+*.wma
+
+# Fonts #
+######################
+*.eot
+*.ttf
+*.woff
+
+# Microsoft Office Files #
+######################
+*.tmp
+*.doc*
+*.xls*
+*.xlk
+*.ppt*
+
+# Backup files #
+#####################
+*.bak
+
+# Misc #
+#####################
+Icon^M^M
+Icon
```

Comment:
Can we put `public` in the `gitignore` as well. We actually don't want to host the output - we will let Netlify do that for us.

Reply:
Similar to how github.com/techqueria/website and github.com/fvcproductions/fvcproductions.github.io does it